.github/workflows/dockerhub-publish.yml

```
@@ -128,15 +182,15 @@ jobs:
         uses: docker/login-action@v1
         with:
           username: ${{ secrets.DOCKER_USERNAME }}
-          password: ${{ secrets.DOCKER_PASSWORD }}
+          token: ${{ secrets.DOCKER_TOKEN }}
 
-      - name: Push ide
+      - name: Push exact version
         run: |
           cd projector-docker
-          ./build-container.sh ${{ secrets.DOCKER_USERNAME }}/${{ matrix.imageName}}:${{ matrix.tag }} ${{ matrix.downloadLink }}
-          docker push ${{ secrets.DOCKER_USERNAME }}/${{ matrix.ImageName }}:${{ matrix.tag }}
+          ./build-container.sh jetbrains/${{ matrix.imageName}}:${{ matrix.tag }} ${{ matrix.downloadLink }}
+          docker push jetbrains/${{ matrix.ImageName }}:${{ matrix.tag }}
       - name: Push latest
         if: ${{ matrix. isLatest }}
         run: |
-            docker tag ${{ secrets.DOCKER_USERNAME }}/${{ matrix.imageName}}:${{ matrix.tag }} ${{ secrets.DOCKER_USERNAME }}/${{ matrix.imageName}}:latest
-            docker push ${{ secrets.DOCKER_USERNAME }}/${{ matrix.ImageName }}:latest
+            docker tag jetbrains/${{ matrix.imageName}}:${{ matrix.tag }} ${{ secrets.DOCKER_USERNAME }}/${{ matrix.imageName}}:latest
```

Let's extract `jetbrains` to one more constant. I've just added `DOCKER_IMAGE_OWNER=jetbrains` secret.

Also, one place is forgotten to be replaced:
```suggestion
            docker tag jetbrains/${{ matrix.imageName}}:${{ matrix.tag }} here/${{ matrix.imageName}}:latest
```